imap_processing/ultra/l0/decom_ultra.py

```diff
@@ -88,11 +88,11 @@ def append_params(decom_data: dict, packet: Packet) -> None:
     ----------
     decom_data : dict
         Dictionary to which the data is appended.
-    packet : space_packet_parser.parser.Packet
+    packet : space_packet_parser.packets.CCSDSPacket
         Individual packet.
     """
-    for key, item in packet.data.items():
-        decom_data[key].append(item.derived_value)
+    for key, item in packet.user_data.items():
+        decom_data[key].append(item.raw_value)
```

Comment:
There is a lot of switching from derived previously to raw now which I think is undesired.

```suggestion
    for key, value in packet.user_data.items():
        decom_data[key].append(value)
```

Reply:
good catch!